### DISCUSSION TITLE
Do you have any plan to use dev containers?

### DISCUSSION THREAD

**Discussion by o4x** [mitmproxy/mitmproxy#5150] [Ideas]
I think it's can be useful for those who want to develop this project in vscode and will not create any issues with other editors.

**Comment by mhils** [comment]
Could you explain what dev containers are or link to some resources? :)

**Comment by o4x** [comment]
With that, we can build a container that dependencies like python and node have already been installed, and also extensions and settings needed to develop easier in vscode are configured. [link](https://code.visualstudio.com/docs/remote/containers) for more details.
I have seen that in other projects like [thefcuk](https://github.com/nvbn/thefuck).

**Reply by mhils** [reply]
That sounds interesting, but I don't think we have any of the main developers use VSCode and would like to maintain it. Please feel free to submit resources via a PR, but it's nothing we're actively seeking out.

**Reply by o4x** [reply]
Ok, I will add that and send a PR to test it.
